internal/pkg/skuba/node/config.go

```diff
@@ -166,7 +170,9 @@ func documentMapToJoinConfiguration(gvkmap map[schema.GroupVersionKind][]byte, a
 		}
 
 		// verify the validity of the YAML
-		strict.VerifyUnmarshalStrict(bytes, gvk)
+		if err := strict.VerifyUnmarshalStrict(bytes, gvk); err != nil {
```

Comment:
Hold this for a second, some of this logic was copied from `kubeadm` helper functions, because the api types were tricky to consume from the outside. Have a look at the comment there: https://github.com/kubernetes/kubernetes/blob/f430a47b606e7863f296603a3d9f6a1655539cf1/cmd/kubeadm/app/util/config/strict/strict.go#L32-L33

Reply:
yeah, we should not check errors on that function 		https://github.com/kubernetes/kubernetes/pull/81736